# Goal
# FISArray
irols/FISArray fis_array

---
# Result
# Success/Fail
bool success

---
# Feedback
float32 height
